pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/Participant.java

```diff
@@ -52,87 +52,89 @@ public class Participant {
     @ColumnName(DBConstants.DDP_PARTICIPANT_ID)
     private String ddpParticipantId;
 
-    @TableName (
+    @TableName(
```

Comment:
was that change made by checkstyle or automatically by intelliJ, can you check if that reformatting is needed? thank you